common/components/ui/Code.scss

```diff
@@ -1,14 +1,42 @@
-pre {
+@import 'common/sass/variables';
+@import 'common/sass/mixins';
+
+code {
+  text-align: left;
   color: #333;
-  background-color: #fafafa;
-  border: 1px solid #ececec;
-  padding: 0.5rem 1rem;
+  background-color: #eff5fe;
+  font-size: 14px;
+  line-height: inherit;
+  border-radius: 2px;
+  padding: 2px 0.25rem;
+  border: 1px solid $border-idle;
+}
+
+pre {
+  font-weight: 400;
+  font-size: 1rem;
+  color: rgba(0, 0, 0, 0.87);
+  background-color: $gray-lightest;
+  border: 1px solid $border-disabled;
+  box-shadow: inset 0 1px 0 0 rgba(63, 63, 68, 0.05);
+  padding: 0.75rem 1rem;
   margin: 0;
+  margin-bottom: 1rem;
   font-size: 1rem;
-  border-radius: 5px;
+  border-radius: 2px;
+  &.wrap {
+    code {
+      white-space: normal;
+    }
+  }
   code {
+    display: block;
+    overflow: scroll;
```

Comment:
Prefer `overflow: auto;` to `overflow: scroll;`. Operating systems that don't have invisible scrollbars (e.g. Windows) will show scrollbars for both vertical and horizontal scrolling all of the time if you force scroll.

I also think it might look nicer to have the scrolling on the `pre` rather than `code`. Right now the scrollbar kind of floats in the middle of the pre.